.gitignore

```diff
@@ -12,375 +12,44 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-## Ignore Visual Studio temporary files, build results, and
-## files generated by popular Visual Studio add-ons.
-##
-## Get latest from https://github.com/github/gitignore/blob/master/VisualStudio.gitignore
+# Poor mans monorepo
+backend/*
```

Comment:
Issue: This is not scoped in api-dh but the root
<img width="159" alt="image" src="https://github.com/user-attachments/assets/6d46dc88-4deb-47f3-9760-af49c9e309aa" />

Reply:
but this is the global .gitignore file, so it is fine right? 🙂